misc/userscripts/qute-keepass

```diff
@@ -148,7 +148,7 @@ def dmenu(items, invocation, encoding):
 
 def get_password():
     """Get a keepass db password from user."""
-    _app = QApplication(sys.argv)
+    QApplication(sys.argv)
```

Comment:
I think we need to keep a handle to this or nothing will work, this change makes this script crash for me.

Reply:
Thx, I re-wrote this commit to put a comment making sure nobody would accidentally remove this variable.